src/line.js

```diff
@@ -1,51 +1,71 @@
 // Line Annotation implementation
-module.exports = function(Chart) {
```

Comment:
It looks like it must have... not sure why github sees it as one big change. But that line was also readded below.